src/find-options/FindOperator.ts

```diff
@@ -1,6 +1,8 @@
 import { ObjectLiteral } from "../common/ObjectLiteral"
 import { FindOperatorType } from "./FindOperatorType"
 import { InstanceChecker } from "../util/InstanceChecker"
+import { ValueTransformer } from "../decorator/options/ValueTransformer"
```

Comment:
Shouldn't be used here.. creates spaghetti

Reply:
@vlad-ostrovski any suggestions how solve this.... ?

Reply:
why not use the same approach already implemented in `SelectQueryBuilder` for `EqualOperator`s?
look one line above your change in `SelectQueryBuilder`

Reply:
@vlad-ostrovski originally, my first approach was to recursively traverse the FindOperators in the parameterValue, but the value of a FindOperator is readonly. Therefore, I had to choose between making it writeable or exposing a transform functionality. Ultimately, I decided to go with the latter since I think it's more readable and provides the added benefit of inheritance (that you also have noted), allowing for different scenarios to be addressed for each operator.